src/main.zig

```diff
@@ -3550,21 +3550,32 @@ fn buildOutputType(
 
             for (std.zig.target.available_libcs) |t| {
                 if (t.arch == target.cpu.arch and t.os == target.os.tag) {
-                    if (t.os_ver) |os_ver| {
-                        std.log.info("zig can provide libc for related target {s}-{s}.{d}-{s}", .{
-                            @tagName(t.arch), @tagName(t.os), os_ver.major, @tagName(t.abi),
-                        });
-                    } else if (t.glibc_min) |glibc_min| {
-                        std.log.info("zig can provide libc for related target {s}-{s}-{s}.{d}.{d}", .{
+                    // If there's a `glibc_min`, there's also an `os_ver`.
+                    if (t.glibc_min) |glibc_min| {
+                        std.log.info("zig can provide libc for related target {s}-{s}.{d}.{d}.{d}-{s}.{d}.{d}", .{
                             @tagName(t.arch),
                             @tagName(t.os),
+                            t.os_ver.?.major,
+                            t.os_ver.?.minor,
+                            t.os_ver.?.patch,
```

Comment:
This is fairly verbose, I'd just pass `t.os_ver.?` in one go - has the added benefit that it would print out even more detailed info if it's ever available.

https://github.com/ziglang/zig/blob/e2e3633612be656177512873a9de6524082d04dd/lib/std/SemanticVersion.zig#L153-L164